web/src/app/admin/admin-message-logs/AdminMessageLogsControls.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react'
+import { Button, Card, Grid } from '@mui/material'
+import ResetIcon from '@mui/icons-material/Replay'
+import FilterIcon from '@mui/icons-material/FilterAlt'
+import { ISODateTimePicker } from '../../util/ISOPickers'
+import Search from '../../util/Search'
+import { useURLParams } from '../../actions'
+import FilterContainer from '../../util/FilterContainer'
+import { ServiceSelect, UserSelect } from '../../selection'
+import { DateTime } from 'luxon'
+
+export default function AdminMessageLogsControls(): JSX.Element {
+  const [params, setParams] = useURLParams({
+    search: '',
+    start: DateTime.now().minus({ hours: 8 }).toISO(),
+    end: DateTime.now().toISO(),
+  })
+
+  const [filterByUser, setFilterByUser] = useState('')
+  const [filterByService, setFilterByService] = useState('')
```

Comment:
It looks like changing the search value and/or filtering by user/service is not re-triggering `useMessageLogs()` to get filtered logs. I believe you mentioned that this story is just adding the graph, so should we disable the search/filtering capabilities for now and address that in a separate story?

Reply:
@tony-tvu After looking at this again, yeah the user and service selects are in local state so nothing ever happens. I'll remove them for this PR so it's easier to review.

Reply:
Removed 👍